test/views/html/employments/UpdateRemoveEmploymentDecisionViewSpec.scala

```diff
@@ -38,22 +38,28 @@ class UpdateRemoveEmploymentDecisionViewSpec extends TaiViewSpec with FormValues
   "update_remove_employment_decision" must {
     behave like pageWithTitle(messages("tai.employment.decision.customGaTitle"))
 
-    behave like pageWithBackLink
+    behave like pageWithBackLinkNew
+
+    "display preheading" in {
+      doc must haveElementAtPathWithText(
+        ".govuk-caption-xl",
+        messages("tai.ptaHeader.accessible.preHeading") + " " + messages("tai.employment.decision.preHeading"))
+    }
 
     "display label with heading" in {
       doc must haveElementAtPathWithText(
-        "#headingInLabel .form-label",
+        ".govuk-fieldset__legend--xl",
         messages("tai.employment.decision.heading", employmentName))
     }
 
     behave like pageWithYesNoRadioButton(
-      UpdateRemoveEmploymentForm.EmploymentDecision + "-yes",
-      UpdateRemoveEmploymentForm.EmploymentDecision + "-no",
+      UpdateRemoveEmploymentForm.EmploymentDecision,
+      UpdateRemoveEmploymentForm.EmploymentDecision + "-2",
```

Comment:
the new library forces the numeric notation: https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukRadios.scala.html#L87

Reply:
is it worth making a pageWithYesNoRadioButton which adds in this -2, it might be used in a few places so could make things tidier

Reply:
I thing this should be left for another case, if the pattern appears again then refactor (although most likely will happen). It's just that i'm not 100% if all cases are going to be similar so once we see them, I'd refactor